### PR TITLE
fda-101 Further updates to accommodate the openfda json documentation.

### PR DESCRIPTION
**Pull request by beardedfinch**

### CODE REVIEW

File: pages/device/classification/reference/_content.yaml [hunk 1]
```diff
@@ -22,7 +22,7 @@
   - review_panel
   - unclassified_reason
   - gmp_exempt_flag
-- "### OpenFDA"
-- "The openFDA section of each document is determined by looking at the product_code value."
+- "### Openfda"
```
Comment (dkrylovsb) [inline pages/device/classification/reference/_content.yaml]: OpenFDA, for consistency.

File: pages/device/pma/reference/_content.yaml [hunk 2]
```diff
@@ -33,6 +33,6 @@
   - zip
   - zip_ext
 - "### Openfda"
```
Comment (dkrylovsb) [inline pages/device/pma/reference/_content.yaml]: OpenFDA, for consistency.

**Comment by dkrylovsb** [comment]
Let's update `### Openfda` to `### OpenFDA` throughout for consistency

**Comment by beardedfinch** [comment]
Added a new commit covering Denis' suggestions.

**Pull request merged**